examples/layers/macos/Runner/Configs/AppInfo.xcconfig

```diff
@@ -0,0 +1,14 @@
+// Application-level settings for the Runner target.
+//
+// This may be replaced with something auto-generated from metadata (e.g., pubspec.yaml) in the
+// future. If not, the values below would default to using the project name when this becomes a
+// 'flutter create' template.
+
+// The application's name. By default this is also the title of the Flutter window.
+PRODUCT_NAME = layers
+
+// The application's bundle identifier
+PRODUCT_BUNDLE_IDENTIFIER = dev.flutter.layers
+
+// The copyright displayed in application information
+PRODUCT_COPYRIGHT = Copyright © 2022 dev.flutter. All rights reserved.
```

Comment:
This looks wrong in most of the other macos examples...
https://github.com/flutter/flutter/blob/e7b7ebc066c1b2a5aa5c19f8961307427e0142a6/dev/integration_tests/flutter_gallery/macos/Runner/Configs/AppInfo.xcconfig#L14

```suggestion
PRODUCT_COPYRIGHT = Copyright 2014 The Flutter Authors. All rights reserved.
```